api/src/main/java/ca/bc/gov/educ/api/gradbusiness/model/dto/v2/School.java

```diff
@@ -0,0 +1,36 @@
+package ca.bc.gov.educ.api.gradbusiness.model.dto.v2;
```

Comment:
This object looks identical to the existing school object, so we don't need it. Can you verify?